### PR TITLE
feature/dashboard:Create Dashboard Model,Dashboard Controller, dashboard table,change folder name from movie to movies and update code

### PR DESCRIPTION
**Pull request by Vinhhtoan**
Create Dashboard Model,Dashboard Controller, dashboard table,change folder name from movie to movies and update code

### CODE REVIEW

File: resources/views/admin/auditoriums/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@extends('layouts.admin')
+@section('content')
+    <div class="flex flex-col items-center justify-center">
+      <h1>Name : {{$auditorium->name}}</h1>
+      <h1>Total Seats : {{$auditorium->total}}</h1>
+    </div>
+@endsection
```
Comment (tcquan-dev) [inline resources/views/admin/auditoriums/show.blade.php]: Thiếu khoảng trắng cuối file

File: resources/views/admin/movies/category/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@extends('layouts.admin')
+@section('content')
+    <div class="flex flex-col items-center justify-center">
+      <h1>Name : {{$category->name}}</h1>
+    </div>
+@endsection
```
Comment (tcquan-dev) [inline resources/views/admin/movies/category/show.blade.php]: Thiếu khoảng trắng cuối file

File: app/Models/Role.php [hunk 3]
```diff
@@ -43,4 +43,30 @@ public function updateRole(array $data)
         $this->fill($data);
         return $this->save();
     }
+
+
```
Comment (tcquan-dev) [inline app/Models/Role.php]: Không được để thừa khoảng trắng

File: resources/views/admin/auditoriums/show.blade.php (hunk 1 above)
Comment (Vinhhtoan) [reply resources/views/admin/auditoriums/show.blade.php]: @tcquan-dev fixed

File: resources/views/admin/movies/category/show.blade.php (hunk 2 above)
Comment (Vinhhtoan) [reply resources/views/admin/movies/category/show.blade.php]: @tcquan-dev fixed

File: app/Models/Role.php (hunk 3 above)
Comment (Vinhhtoan) [reply app/Models/Role.php]: @tcquan-dev fixed

File: resources/views/admin/auditoriums/show.blade.php (hunk 1 above)
Comment (tcquan-dev) [reply resources/views/admin/auditoriums/show.blade.php]: @Vinhhtoan ok

File: resources/views/admin/movies/category/show.blade.php (hunk 2 above)
Comment (tcquan-dev) [reply resources/views/admin/movies/category/show.blade.php]: @Vinhhtoan ok

File: app/Models/Role.php (hunk 3 above)
Comment (tcquan-dev) [reply app/Models/Role.php]: @Vinhhtoan ok

**Pull request merged**